### PR TITLE
Change log for 6.6.1

### PR DESCRIPTION
**Pull request by alexanderzobnin**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,28 @@ backendSrv.get(‘http://your.url/api’).then(result => {
 
 You can test your plugin with the `master` branch version of Grafana.
 
+# 6.6.1 (2020-02-06)
+
+### Features / Enhancements
+* **Quota**: Makes LDAP + Quota work for the first login of a new user. [#21949](https://github.com/grafana/grafana/pull/21949), [@xlson](https://github.com/xlson)
+
+### Bug Fixes
+* **Annotations**: Change indices and rewrites annotation find query to improve database query performance. [#21915](https://github.com/grafana/grafana/pull/21915), [@papagian](https://github.com/papagian)
```
Comment (papagian) [inline CHANGELOG.md]: can you add co-authors?
[@marefr](https://github.com/marefr), [@kylebrandt](https://github.com/kylebrandt)

Comment (marefr) [reply CHANGELOG.md]: Doesn't matter :smile:

File: CHANGELOG.md [hunk 2]
```diff
@@ -21,6 +21,28 @@ backendSrv.get(‘http://your.url/api’).then(result => {
 
 You can test your plugin with the `master` branch version of Grafana.
 
+# 6.6.1 (2020-02-06)
+
+### Features / Enhancements
+* **Quota**: Makes LDAP + Quota work for the first login of a new user. [#21949](https://github.com/grafana/grafana/pull/21949), [@xlson](https://github.com/xlson)
```
Comment (marefr) [inline CHANGELOG.md]: This one should be under bug fixes

**Review by marefr — CHANGES_REQUESTED** [review]
See comment

**Review by papagian — APPROVED** [review]
LGTM!

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**